ISSUES.md

```diff
@@ -10,6 +10,20 @@ Detailed information about each Bundler command, including help with common prob
 
 ## Troubleshooting
 
+### Permission denied when installing bundler
+Certain operating systems such as MacOS and Ubuntu have versions of Ruby that requires evelated privileges to install gems.
```

Comment:
hi @segiddins, not sure if you're not seeing but the latest commit already fixed these issues.